ci/settings/master-branch.yml

```diff
@@ -0,0 +1,13 @@
+project: concourse-pypi-resource-pipeline-template
+docker_tag: latest
+docker_tag_add_file: version/date.txt
+github_tag_add_file: version/name.txt
+branch: master
+trigger: true
+pci-gchat_hook: ((gchat_hook_integration_team))
+
+dependency-updater:
```

Comment:
I would not do that. First because we will always update all dependencies manually. Secondly, you disabled, in your previous PR, Github dependabot that worked fine for public dependencies..

Reply:
Done, removed